services/utils/formatDate.js

```diff
@@ -0,0 +1,8 @@
+export function formatDate(date, locale = "en-GB") {
```

Comment:
The defaults we discussed can be added here

Reply:
Sure I will add it, sorry I am a bit busy I will end it asap.

Reply:
No problem, when you get time - I only asked because you kept chasing me to merge your PR 😂